packages/block-editor/src/components/rich-text/index.js

```diff
@@ -763,6 +781,7 @@ export class RichText extends Component {
 		// In all other cases, prevent default behaviour.
 		event.preventDefault();
 
+		const formats = createPrepareEditableTree( this.props )( value );
```

Comment:
I don't understand why this change is here.

Reply:
I'll comment, it's to make sure the boundaries work for annotations.